docs/source/developer_guide/index.rst

```diff
@@ -1,17 +0,0 @@
-Practical information concerning how to set up your development environment and how
-to contribute to aiida core can be found on the 
-`AiiDA Wiki <https://github.com/aiidateam/aiida-core/wiki>`_.
```

Comment:
is this information still present at a prominent position?

Reply:
No, unfortunately not.
However, I think perhaps it should be moved to the front page, under a special section "For developers"?

Reply:
Also note that this particular page was basically unreachable before - unless somebody pressed the `<no title>` "breadcrumb".

Reply:
could you please preserve this information in a place where it makes sense?